techniques/html/H85.html

```diff
@@ -51,44 +51,40 @@
 
 <div class="default-grid with-gap leftcol">
 <aside class="box nav-hack sidebar standalone-resource__sidebar">
-    <header class="box-h">Page Contents</header>
+    <header class="box-h">このページの内容</header>
     <div class="box-i">
       <nav aria-label="page contents" class="navtoc">
-        <ul><li><a href="#technique">About this Technique</a></li>
-<li><a href="#description">Description</a></li>
-<li><a href="#examples">Examples</a></li>
-<li><a href="#resources">Related Resources</a></li>
-<li><a href="#tests">Tests</a></li>
+        <ul><li><a href="#technique">このテクニックについて</a></li>
+<li><a href="#description">解説</a></li>
+<li><a href="#examples">事例</a></li>
+<li><a href="#resources">関連リソース</a></li>
+<li><a href="#tests">検証</a></li>
 </ul>
       </nav>
     </div>
 </aside>
 
 <main id="main" class="standalone-resource__main">
     
-<h1><span>Technique H85:</span>Using <code class="language-html">optgroup</code> to group <code class="language-html">option</code> elements inside a <code class="language-html">select</code></h1>
+<h1><span>テクニック H85:</span><code class="language-html">select</code> 要素内の <code class="language-html">option</code> 要素をグループ化するために、<code class="language-html">optgroup</code> 要素を使用する</h1>
 
 
 
 
 <section id="technique" class="box">
-	<h2 class="box-h box-h-icon">About this Technique</h2>
+	<h2 class="box-h box-h-icon">このテクニックについて</h2>
 	<div class="box-i">
 
 
   
   <p>
-    This technique relates to
-    
-<a href="https://w3c.github.io/wcag/understanding/info-and-relationships">1.3.1: Info and Relationships</a>
-(<strong>Sufficient</strong>
-  when used for making information and relationships conveyed through presentation programmatically
-                     determinable).</p>
+    このテクニックは、<a href="https://waic.jp/translations/WCAG22/Understanding/info-and-relationships">1.3.1: 情報及び関係性</a>
+(提示を通じて伝えられる情報及び関係性をプログラムによる解釈を可能にするために用いられる場合は<strong>十分なテクニック</strong>) に関連する。</p>
   
 
 
 
-  <p>This technique applies to <abbr title="HyperText Markup Language">HTML</abbr> pages that collect user input.</p>
+  <p>このテクニックは、利用者の入力を収集する <abbr title="HyperText Markup Language">HTML</abbr> ページに適用する。</p>
```

Comment:
他のテクニックのイントロダクションと揃えて、「〜に適用される」としました。
```suggestion
  <p>このテクニックは、利用者の入力を収集する <abbr title="HyperText Markup Language">HTML</abbr> ページに適用される。</p>
```